overlord/snapstate/snapmgr.go

```diff
@@ -330,14 +330,14 @@ func updateInfo(st *state.State, snapst *SnapState, channel string, userID int,
 	return res[0], nil
 }
 
-func snapInfo(st *state.State, name, channel string, userID int, flags Flags) (*snap.Info, error) {
+func snapInfo(st *state.State, name, channel string, rev snap.Revision, userID int, flags Flags) (*snap.Info, error) {
```

Comment:
Yeah, that originated back when were getting rid of /snappy madness, and I didn't want to bother Samuele with such trivials when there are a world of changes coming, but made the same point there: let's try to be consistent when we can and that's cheap.